compat/register-hooks.php

```diff
@@ -91,6 +91,7 @@
 add_action( 'notification/init', [ $this->component( 'integration_wp_emails' ), 'disable_password_change_notify_to_admin' ], 10, 0 );
 add_action( 'notification/init', [ $this->component( 'integration_wp_emails' ), 'disable_send_confirmation_on_profile_email' ], 10, 0 );
 add_action( 'notification/init', [ $this->component( 'integration_wp_emails' ), 'disable_send_confirmation_on_admin_email' ], 10, 0 );
+add_action( 'notification/init', [ $this->component( 'integration_wp_emails' ), 'disable_send_confirmation_on_admin_email_changed' ], 10, 0 );
```

Comment:
In this place i have two error on command "composer phpstan". In line 92 and 94:
`Ignored error pattern #^Parameter \#2 \$callback of function add_action expects callable\(\)\: mixed, array\(mixed, '\S+'\) given\.$# in path... register-hooks.php is expected to occur 81 times, but occurred only 1 time`
Is this error significant?

Reply:
Ohh, nice catch with this file! You don't have to bother with that, though :)

It's generated upon the release, as it's just the compatibility dump. The `@action` comment is enough for development purposes and in most production cases

Reply:
And the error is not much significant, if that's the only one you can run `composer phpstan -- --generate-baseline` to get rid of it